docker/Dockerfile.alpine

```diff
@@ -0,0 +1,33 @@
+FROM alpine:3.14.3
+LABEL maintainer="Vincenzo Palazzo (@vincenzopalazzo) vincenzopalazzodev@gmail.com"
+
+
+WORKDIR /build
+
+RUN apk update && \
+    apk add ca-certificates alpine-sdk autoconf automake git libtool \
+    gmp-dev sqlite-dev python3 py3-mako net-tools zlib-dev libsodium gettext su-exec \
+    python3 py3-pip && \
+    apk add --upgrade fortify-headers
+
+RUN mkdir lightning
```

Comment:
No need to first `mkdir` the copy below should suffice.

Reply:
Mh, the COPY command should not create a directory, or I-m missing some things?